### PR TITLE
Use prev epoch source naming correctly

### PR DESCRIPTION
**Pull request by terencechain**
In Altair `previous_epoch_attester`'s naming does not reflect its intend. What it really means is `previous_epoch_source_attester`. This PR updates it to avoid further confusion down the line

Note I did not update anything for phase0, because phase0 is still technically correct where source == included, I also want to avoid changing consensus logic as much as possible

### CODE REVIEW

**Comment by terencechain** [comment]
@nisdas

File: beacon-chain/core/epoch/precompute/attestation.go [hunk 1]
```diff
@@ -13,14 +13,15 @@ import (
...
 	"github.com/prysmaticlabs/prysm/proto/prysm/v1alpha1/attestation"
+	"github.com/prysmaticlabs/prysm/runtime/version"
 	"go.opencensus.io/trace"
 )
 
 // ProcessAttestations process the attestations in state and update individual validator's pre computes,
 // it also tracks and updates epoch attesting balances.
 func ProcessAttestations(
 	ctx context.Context,
-	state state.ReadOnlyBeaconState,
```
Comment (prestonvanloon) [inline beacon-chain/core/epoch/precompute/attestation.go:23]: Why change from ReadOnly state? Does this method now write or mutate the state?

Comment (terencechain) [reply beacon-chain/core/epoch/precompute/attestation.go:23]: `state.Version()` is not defined in ReadOnly for some reason. Probably the correct thing to do is to move `Version()` to ReadOnly instead. Thanks for the comment

Comment (prestonvanloon) [reply beacon-chain/core/epoch/precompute/attestation.go:23]: Yes, I agree with that assessment. Move Version to the ReadOnly interface

Comment (terencechain) [reply beacon-chain/core/epoch/precompute/attestation.go]: ready again @prestonvanloon

**Review by prestonvanloon — APPROVED** [review]

**Pull request merged**